docs/quickstart.md

```diff
@@ -0,0 +1,119 @@
+# Quickstart
+
+This page describes how to run, test, and develop PrairieLearn. Ensure that you have already installed PrairieLearn either:
+
+- Natively, by following the [running natively](installingNative.md) guide
+- Via Docker, by following the [running via Docker](installingLocal.md) guide
+
+## Development server
+
+- Run the server in development mode to automatically restart when changes are detected:
+
+  ```sh
+  make dev
+  ```
+
+- In a web-browser go to [http://localhost:3000](http://localhost:3000).
+
+- To stop the server, use ++ctrl+c++.
+
+## Production server
+
+You can build and run the code to more closely mimic what will happen in production environments:
+
+```sh
+make build
+make start
+```
+
+## Workspaces
+
+- If you need support for [workspaces](workspaces/index.md), ensure Docker is installed and running, and then in a separate terminal run:
+
+```sh
+sudo make dev-workspace-host # or sudo make start-workspace-host
+```
+
+## Documentation
+
+If you want to preview the documentation, run:
+
+```sh
+make preview-docs
+```
+
+## Testing
+
+If you are contributing code to PrairieLearn, you must ensure that your changes work and pass our style guidelines. More information on debugging and testing can be found on the [developer guide](./dev-guide/index.md). Information on contributing can be found on the [contribution guide](./contributing.md).
+
+- Run the test suite (Docker must be installed and running):
+
+```sh
+make test
+```
+
+Or, more specifically:
+
+```sh
+make test-js     # Javascript only
+make test-python # Python only
+```
+
+The main JavaScript test suite can take on the order of 10+ minutes to run, so you may only run specific test files as you develop. To run specific test files, you first need to run `make start-support` to start the database and other services, and then you can run specific test files.
```

Comment:
Maybe make a note about package-specific tests (e.g. `cd packages/csv && yarn test`)? This came up with James from Walla Walla recently.

Reply:
Added in 11571eaf4cb629f088c608b49baefe0e835027bd.